openslides_backend/action/action.py

```diff
@@ -534,8 +535,8 @@ def execute_other_action(
         to the called class if set. Usually this is needed for cascading deletes from
         outside of meeting.
         """
-        if hasattr(self.__class__, "skip_archived_meeting_check"):
-            skip_archived_meeting_check = self.__class__.skip_archived_meeting_check
```

Comment:
@r-peschke: I'm not sure if this might have unintended side effects, but from my understanding, if an action was executed without this check, all dependent actions should also skip this check and not only the ones one "level" below it (that was the case before). At least for `meeting.delete` -> `projector.delete` -> `projection.delete` this is needed and makes it much easier this way.

Reply:
The intention is identical, but the former implementation is wrong. It only check for existence of the attribute, but overwrites the method-parameter, which is wrong. Therefore I think this is the intended behavior.

Reply:
perfekt